4.3.2 release notes (#17)
128technology/docs: docs/release_notes/128t_release_notes_4.3.2.md

## Issues Fixed
- **I95-33559** `send command download router all <version>` will report `% Error: Failed to execute the 'download-software' RPC: Request failed with error: TIMEOUT` even though the command completes successfully.

  _**Conditions:**_ Execute download command from Conductor with >25 managed assets.

- **I95-33655** GUI Ping and Service Ping do not report incremental ICMP response messages; only the summary is produced when the command has completed.

  Until the system is upgraded to 4.3.2, this issue can be mitigated by using the PCLI versions of the same command.

- **I95-33759** GUI DHCP Client Lease table reports no leases while DHCP server has active client leases.
  Until the system is upgraded to 4.3.2, this issue can be mitigated by using the PCLI version of the same command: `show network-interface application node <node-name> name <interface-name>`

- **I95-33789** Forwarding core count incorrectly reported in GUI under Router -> Node -> Platform Information
  Until the system is upgraded to 4.3.2, this issue can be mitigated by using the PCLI version of the same command: `show platform cpu `

> - **I95-33938** REST API missing documentation reference to command to delete configuration backups.  The missing, and supported, API is  `DELETE /api/v1/config/export/:filename`

Reviewer: consider rewording (to to for):

REST API missing documentation reference for the command to

Also should the comma be removed here?

The missing, and supported,

- **I95-33979** Custom Reports in the GUI created with peer path metrics for jitter, latency, or loss are rendered as a line chart instead with a single value instead of one per path

  _**Conditions:**_ Custom Reports rendering data from peer paths when multiple peer paths exist

- **I95-34015** nginx process does not properly restart during a rollback of 128T

  _**Symptom:**_ systemd continually attempts to restart nginx process

  _**Conditions:**_ 128T software is being downgraded to version 4.2.0 <= version <= 4.2.4

- **I95-34061** Linux command equivalent of the PCLI `connect` command: `connect128t --router <router-name> --node <node name>` produces the cosmetic error:
  
  ```
  /usr/lib/128technology/par/connect128t.par/subpar/runtime/support.py:117: UserWarning: Unable to extract to requested directory '/usr/lib/128technology/unpar/connect128t', falling back to tmp dir
  ```
  There is no impact to the functionality of `connect128t`

- **I95-34068** SVR sessions fail to establish due to waypoint allocation failures after HA node failover.

  _**Symptom:**_ The following warning log is generated:

>   

Reviewer: similar warning message provided as the IP addresses may be different. 

This also bring up an interesting aspect. We need to be very careful when using addresses and make sure that they are all private addresses and nothing public. General rule.

  ```

> Mar 03 09:25:10.813 [HWMC| – ] WARN (icmpManager ) Base Exception: failed to allocate ports for WayPoint; intf=5.0; local=172.27.233.47; remote=10.61.55.109

Reviewer 2: ```suggestion
Mar 03 09:25:10.813 [HWMC| – ] WARN (icmpManager ) Base Exception: failed to allocate ports for WayPoint; intf=5.0; local=192.0.2.100; remote=198.51.100.128
```

  ```

  Until the system is upgraded to 4.3.2, this issue can be mitigated by removing the corresponding adjacency configuration and adding it back.
  
- **I95-34164** Load balancer occasionally returns standby paths during packet duplication flow setup
  
- **I95-34249** Time scale selector in upper right hand of Top 10 Sessions only shows the last hour of data, regardless of what value is selected.

- **I95-34371** Large FIBs can take excess of 30 seconds to load, causing the highway process to fault and restart

  _**Symptom:**_ System would appear to take a long time to load its FIB

  _**Conditions:**_ System running at TRACE log level may induce this problem

  Until the system is upgraded to 4.3.2, this issue can be mitigated by ensuring the system is configured at log level INFO, and if necessary, reduce the size of the service configuration.

> - **I95-34397** ARP caching and reverse mac learning is not triggered when ARP response takes longer than 1 second

Reviewer 2: ```suggestion
- **I95-34397** ARP caching and reverse MAC learning is not triggered when an ARP response takes longer than 1 second
```

> - **I95-34421** Lenovo onboard Intel X722 NIC is not recognized by 128T and can not used as a forwarding interface

Reviewer 2: ```suggestion
- **I95-34421** Lenovo onboard Intel X722 NIC is not recognized by 128T and cannot be used as a forwarding interface
```

- **I95-34498** Erroneous centos-release-yum4 errors in `/var/log/salt/minion` log file

  _**Symptom:**_ Log messages appear in salt minion file

  ```
2020-03-25 17:59:37,893 [salt.loader.12.230.70.246.int.module.cmdmod][ERROR ][18140] stdout: package centos-release-yum4 is not installed
  ```

  _**Conditions:**_ Automated Provisioner performs a software version query 
  

> - **I95-35477** Interface never becomes active when `shared-physical-address` is configured to be the same as the physical MAC

Reviewer 2: ```suggestion
- **I95-35477** Interface never becomes active when `shared-phys-address` is configured to be the same as the physical MAC
```

- **I95-34645** Swagger API for "clone" and "move" operations are incorrect.  They are `/config/{configStore}/authority/district/{district}/clone` when they should be `/config/{configStore}/authority/district/clone`

- **I95-34653** SNMP IF-MIB does not display correctly when a non-forwarding management interface is present

> 

Reviewer: entry needed here:

- **I95-33845** security package updates
